spec/lib/mt9/credit_batch_spec.rb

```diff
@@ -32,13 +32,13 @@
         name: "This Name",
         code: "This Code",
         alpha_reference: "This Alpha",
-        particulars: "This Particulars",
+        particulars: "This Particulars"[0...12],
```

Comment:
I'm a little concerned about maintaining these `[0...12]` inline transforms on string literals throughout our specs. It might be better to set them to valid literal values as it looks like you've added appropriate coverage for the "too long" context. WDYT?

Reply:
Yeah that might be better. My thinking was that it could be example usage of how you'd transform the strings into their correct lengths, but it's inside a spec so I think we'll just actually make them the correct length